### PR TITLE
K8S-545 Octavia MaaS addendum

### PR DESCRIPTION
**Pull request by xgerman**
This adds public API checks

### CODE REVIEW

File: playbooks/templates/rax-maas/lb_api_check_octavia.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{% set label = "lb_api_check_octavia" %}
+{% set check_name = label+'--'+maas_lb_name %}
+type              : remote.http
+label             : "{{ check_name }}"
+period            : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
+timeout           : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
+disabled          : "{{ (inventory_hostname != groups['octavia_api'][0] or check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
```
Comment (npawelek) [inline playbooks/templates/rax-maas/lb_api_check_octavia.yaml.j2]: Is the group used here `octavia_api` or `octavia-api`? I noticed the group in the `maas-openstack-octavia.yml` playbook was calling `octavia-api` specifically.

**Comment by npawelek** [comment]
@xgerman thanks! Just one other thing I noticed, but was unable to add into your branch. Can you remove 'private_lb_api_check_octavia.yaml.j2' from the top level directory of the repo?

**Comment by xgerman** [comment]
@npawelek I think the remove shows up as a rename...

**Comment by npawelek** [comment]
@xgerman Looks like the `lb_api_check` is good now, but I made some additions to the private check in #386 (https://github.com/rcbops/rpc-maas/pull/386/files#diff-a2f9553b3ef6d44aab610925ba1fdbe6) so we'll want to make sure it remains aligned if changes are being committed.

**Comment by xgerman** [comment]
I rebased so I think we got everyhting?

**Comment by npawelek** [comment]
recheck_all

**Comment by npawelek** [comment]
recheck_cit_all

**Comment by xgerman** [comment]
recheck_cit_all

File: playbooks/maas-openstack-octavia.yml [hunk 2]
```diff
@@ -55,8 +55,18 @@
...
+      template:
+        src: "templates/rax-maas/lb_api_check_octavia.yaml.j2"
+        dest: "/etc/rackspace-monitoring-agent.conf.d/lb_api_check_octavia.yaml"
+        owner: "root"
+        group: "root"
+        mode: "0644"
+      delegate_to: "{{ physical_host | default(ansible_host) }}"
+      when:
+        - maas_remote_check | bool
+        - not maas_private_monitoring_enabled
```
Comment (tonytan4ever) [inline playbooks/maas-openstack-octavia.yml]: Need a line separator.

**Comment by tonytan4ever** [comment]
Just one minor issue, and need rebase from master. Otherwise LGTM.

**Comment by tonytan4ever** [comment]
LGTM, all gate check passed.

**Pull request merged**